shared/src/main/scala/amf/plugins/document/graph/parser/ExpandedGraphParser.scala

```diff
@@ -55,7 +53,7 @@ class ExpandedGraphParser()(implicit val ctx: GraphParserContext)
       val maybeMaps = document.node.toOption[Seq[YMap]]
       val maybeMap  = maybeMaps.flatMap(s => s.headOption)
       val maybeMaybeObject = maybeMap.flatMap(map => {
```

Comment:
Lines 53, 54 and part of 55 could be compacted into one function call to avoid overhead of how we extract the Map from the Document